### PR TITLE
Move astro helper functions to their own file

### PR DESCRIPTION
**Pull request by jeremykubica**
Move the astronomy helper functions from `analysis_utils.py` and `run_search.py` to a single file for astronomy helper functions. As a bonus there were two versions of `_calc_barycentric_corr` one of which was never called. This removes the uncalled version.

This is step one in a more comprehensive refactoring of analysis_utils to break out file, astronomy, and filtering functions.

Note the two functions don't currently have tests. Issues #229 and #228 created to track that gap.

### CODE REVIEW

**Review by cchris28 — COMMENTED** [review]
Does this affect README.md or any of the notebooks?

**Comment by jeremykubica** [comment]
No. The notebooks use these functions through run_search.py instead of calling them directly. So moving them should not impact anything.

File: src/kbmod/astro_utils.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    radius of dist au. This function returns a linear fit to the barycentric
+    correction as a function of position on the first image.
+
+    Parameters
+    ----------
+    img_info : `kbmod.search.ImageInfo`
+        ImageInfo
+    dist : `float`
+        Distance to object from barycenter in AU.
+    """
```
[the remark this answers is not in the document]
Comment (jeremykubica) [reply src/kbmod/astro_utils.py]: Added.

File: src/kbmod/run_search.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 import numpy as np
 from .analysis_utils import Interface, PostProcess
+from .astro_utils import *
```
[the remark this answers is not in the document]
Comment (jeremykubica) [reply src/kbmod/run_search.py]: This is a good point. Looking at this some more, the functions aren't really general purpose. They are very specific to run_search.py. I moved them in there (or back in there for the barycentric corrections) for now.

**Review by cchris28 — APPROVED** [review]

**Pull request merged**